tests/unit_tests/test_server.py

```diff
@@ -0,0 +1,108 @@
+"""Unit tests for the SkyPilot API server."""
```

Comment:
Nice! This is great. Let's move this unit_tests to the specific subfolder, i.e. `unit_tests/sky/server/test_server.py`?

Reply:
Can we keep directories flat?